language/stdlib/modules/lcs.move

```diff
@@ -0,0 +1,7 @@
+// Utility for serializing a Move type to its binary LCS representation
+
+address 0x0:
+module LCS {
```

Comment:
As mentioned in the meeting, we have address aliasing, so I'd prefer a less obtuse name. `LibraCanonicalSerialization` seems a lot more helpful

Reply:
I am sympathetic to this (I also hate abbrevs), but am leaning toward sticking with `LCS` given that it's the name we use everywhere in the Rust codebase and our written communication (similar to TLS, not `TransportLayerSecurity`...)

Reply:
I think the abbreviation is a bit more palatable in the Rust code as it is a much more well known and integral algorithm. "If you don't know what LCS is and you are in this part of the Rust code base, what are you doing"

But I don't really feel that the same holds for a given Move developer

Reply:
I think that a Libra Move developer probably would need to know LCS (e.g., to encode transactions, arguments to scripts, etc.)? For example: Eth's [RLP](https://github.com/ethereum/wiki/wiki/RLP) is pretty important for Solidity/EVM programmers to know (that would have been a better analogy than TLS).